app/controllers/recommended_feeds_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RecommendedFeedsController < ApplicationController
+  include PageableRecommendedFeedConcern
+
+  before_action :authenticate_user!
+
+  def show
+    respond_to do |format|
+      format.html { initialize_pageable_recommended_feed }
+      format.js {
+        @feed_item = RecommendedFeedItemPresenter.new(
+          feed_item: current_user.toggle_feed_item_recommended(params[:id]),
+          current_user: current_user,
+        )
+      }
+    end
+  end
+end
```

Comment:
Simple, maintainable. Postbacks result from page navigation - pull ONLY those recommended feeds we need to appear in the UI, NOT the whole database with every pull. Ajax calls simply need to retrieve the feed item that is being UNrecommended (UNstarred); unobtrusive javascript allows us to only update that SINGLE html element in the UI, making for a much better user experience.